intrastat_product/models/account_move.py

```diff
@@ -160,11 +169,23 @@ class AccountMoveIntrastatLine(models.Model):
     transaction_weight = fields.Integer(
         help="Transaction weight in Kg: Quantity x Product Weight"
     )
+    # product_origin_country_id is replaced by product_origin_country_code
+    # this field should be dropped once the localisation modules have been
+    # adapted accordingly
     product_origin_country_id = fields.Many2one(
         comodel_name="res.country",
         string="Country of Origin",
         help="Country of origin of the product i.e. product " "'made in ____'.",
     )
+    product_origin_country_code = fields.Char(
+        string="Country of Origin of the Product",
+        size=2,
+        required=True,
+        default="QU",
```

Comment:
I find it a bit "too easy" to use QU by default. I think we should display a warning in the warning dialog box about the fact that the country of origin is missing on product X and say that we've set the country code to QU as a temporary solution.